lib/rubocop/cop/rspec/capybara/current_path_expectation.rb

```diff
@@ -103,11 +103,22 @@ def add_ignore_query_options(corrector, node)
             expectation_last_child = expectation_node.children.last
             return if %i[regexp str].include?(expectation_last_child.type)
 
+            comma = ',' if expectation_last_child_comma?(expectation_node,
+                                                         expectation_last_child)
+
             corrector.insert_after(
               expectation_last_child,
-              ', ignore_query: true'
+              "#{comma} ignore_query: true"
             )
           end
+
+          def expectation_last_child_comma?(node, last_child)
+            trailing_config = config.for_cop('Style/TrailingCommaInArguments')
+            return true unless trailing_config['Enabled']
+
+            trailing_config['EnforcedStyleForMultiline'] == 'no_comma' ||
```

Comment:
I think this is too tight a coupling between RuboCop-RSpec and RuboCop’s internals.

Can we get away with just defining an `.autocorrect_incompatible_with` method instead?